src/commands/core/plugins/add.ts

```
@@ -1,11 +1,12 @@
 import { Args, Flags } from '@oclif/core'
 
-import { addCollectionPlugin, AddCollectionPluginArgsPlugin, addPlugin, AddPluginArgsPlugin, fetchAsset } from '@metaplex-foundation/mpl-core'
+import { addCollectionPlugin, AddCollectionPluginArgsPlugin, addPlugin, AddPluginArgsPlugin } from '@metaplex-foundation/mpl-core'
```

ESLint

[error] Expected "`@metaplex-foundation/mpl-core`" to come before "`@oclif/core`"

(perfectionist/sort-imports)